android/app/src/main/java/com/zulipmobile/sharing/SharingHelper.kt

```diff
@@ -25,16 +25,16 @@ fun maybeHandleIntent(
     contentResolver: ContentResolver
 ): Boolean {
     // We handle intents from "sharing" something to Zulip.
-    if (intent?.action == Intent.ACTION_SEND) {
-        handleSend(intent, application, contentResolver)
-        return true
+
+    when (intent?.action) {
+        Intent.ACTION_SEND -> handleSend(intent, application, contentResolver)
+        Intent.ACTION_SEND_MULTIPLE -> handleSend(intent, application, contentResolver)
```

Comment:
```suggestion
        Intent.ACTION_SEND, Intent.ACTION_SEND_MULTIPLE ->
            handleSend(intent, application, contentResolver)
```

This makes it a bit clearer that these bodies are indeed the same.